src/main/java/com/codeborne/selenide/ex/ErrorMessages.java

```diff
@@ -39,7 +39,15 @@ public static String screenshot(String screenshotPath) {
     if (!Configuration.screenshots) {
       return "";
     }
-    return "\nScreenshot: " + screenshotPath;
+    if (Configuration.savePageSource) {
```

Comment:
if Configuration.screenshots=false and Configuration.savePageSource=true
we will not have PageSource in error message. I guess it's wrong.

Reply:
Never mind. I look through code and understand why it is so.